src/lib/validations/utils.ts

```diff
@@ -35,3 +40,27 @@ export function getMultilineStatement( statementRegex: RegExp ): ( line: string
 		return matchingStatements;
 	};
 }
+
+export function getFileType( filepath: string ): string {
+	const regex = new RegExp( /^\w+\/[-+.\w]+/g );
+
+	let result;
+
+	try {
+		result = cp.execFileSync( '/usr/bin/file', [ '-b', '--mime-type', filepath ] );
```

Comment:
⚠️ I am not sure about the windows support for this ⚠️

Reply:
I don't see a Windows native way to do this either. The best bet is to rely on a third party program [`GnuWin32's file`](https://gnuwin32.sourceforge.net/packages/file.htm)

Reply:
For now, could we conditionally run this if the file command exists?

Reply:
In this way we could even support GnuWin32 file based on the OS platform.

Reply:
I am not even sure about proper Linux support :-) On a typical Linux desktop environment, `file` is not installed by default.

We need to check if `file` exists and not bail out if it does not or cannot be executed.

Reply:
Added a check to see if the problem exists on computer. If not, the mimetype check is bypassed.